docs/CodeTransformation.md

```diff
@@ -22,29 +22,68 @@ If you override the `transform` configuration option `babel-jest` will no longer
 You can write you own transformer. The API of a transformer is as follows:
 
 ```ts
-interface Transformer<OptionType = unknown> {
-  /**
-   * Indicates if the transformer is capabale of instrumenting the code for code coverage.
-   *
-   * If V8 coverage is _not_ active, and this is `true`, Jest will assume the code is instrumented.
-   * If V8 coverage is _not_ active, and this is `false`. Jest will instrument the code returned by this transformer using Babel.
-   */
```

Comment:
should probably keep this. If it's missing from the code (which I assume) we should add it there

Ideally this'd be kept automatically in sync by something like #11657

Reply:
ok. added it to the code as well